src/graphql/types/object/wallet-contact.ts

```diff
@@ -9,7 +9,7 @@ import Username from "../scalar/username"
 
 import { TransactionConnection } from "./transaction"
 
-const AccountContact = new GT.Object({
+const AccountContact = GT.Object<UserRecord, GraphQLContextForUser>({
```

Comment:
shouldn't `GraphQLContextForUser` be the third argument?

Reply:
Nope. You don't define the args for resolvers here. We can use `GT.Field` to define args for a field, which we're doing for the mutations, but not queries (yet)